packages/logos-docusaurus-theme/src/client/components/mdx/Grid/Grid.tsx

```diff
@@ -81,6 +108,33 @@ const GridRoot = styled.div<{
     gap: 0 1rem;
   }
 
+  .mdx-grid__scroll--space-between {
+    width: 100%;
+
+    > div {
+      justify-content: space-between;
+      width: 100%;
+    }
+  }
+
+  .mdx-grid__scroll__button {
+    width: 83px;
+    gap: 0.75rem;
+    align-items: center;
+    align-self: stretch;
+    border: 1px solid rgb(var(--lsd-border-primary)) !important;
+  }
+
+  .mdx-grid__scroll__button--left {
+    padding: 6px 10px 6px 12px;
+    justify-content: flex-start;
+  }
+
+  .mdx-grid__scroll__button--right {
+    padding: 6px 10px 6px 12px;
+    justify-content: flex-end;
+  }
```

Comment:
I noticed the scroll buttons are added to the mobile design as well:

https://www.figma.com/file/TFbzcQ1N622YGMMzdPQr6B/UI?type=design&node-id=4027-17529&mode=dev

The team component on Figma doesn't have them on the mobile version, so it should be optional. I'd suggest adding a new property `scrollButtons?: boolean` to the `GridBreakpointProps`.